### ISSUE TITLE
Add design for Login page

### ISSUE THREAD

**Issue by Musali** [phuens/JobList#2]
Make a better design for login sign up and sign in

**Comment by Musali** [comment]
@phuens @cbjoachi Some design ideas:

<img width="775" alt="Screen Shot 2019-11-20 at 11 43 05 AM" src="https://user-images.githubusercontent.com/13123893/69259246-f3640700-0b8b-11ea-85bd-7c1033d90b2f.png">

<img width="775" alt="Screen Shot 2019-11-20 at 11 43 28 AM" src="https://user-images.githubusercontent.com/13123893/69259248-f3640700-0b8b-11ea-8190-91803dad74ef.png">

**Comment by phuens** [comment]
The top and bottom part are separate right ? @Musali

**Comment by Musali** [comment]
Yeah one is for sign up, and the first is an idea of what users see when they first come to the site.

Idk if it should be a whole page or a smaller modal in the center.
